addons/website_sale/models/sale_order.py

```diff
@@ -21,6 +23,10 @@ class SaleOrder(models.Model):
     )
     cart_quantity = fields.Integer(compute='_compute_cart_info', string='Cart Quantity')
     only_services = fields.Boolean(compute='_compute_cart_info', string='Only Services')
+    is_abandoned_cart = fields.Boolean(compute='_compute_abandoned_cart', string='Abandoned Cart')
+    cart_recovery_email_sent = fields.Boolean(
+        default=False,
```

Comment:
As this is the default value of all boolean fields there is no need to repeat it. It simplifies the model declaration.